LeadKitAdditions/.swiftlint.yml

```diff
@@ -1,8 +1,9 @@
 disabled_rules:
-  - variable_name
+  - identifier_name
```

Comment:
looks like copy-paste. should be link to submodule file. see https://github.com/TouchInstinct/LeadKit/pull/73/files#diff-e9e9dfc22d31913c2e705d4d571d54c7R1

<img width="572" alt="screen shot 2017-07-28 at 11 21 54" src="https://user-images.githubusercontent.com/6436245/28709017-1b2303d0-7387-11e7-9506-a3e887606fba.png">